Various fixes and 1.8 Prerelease block updates.

Khroki opened the pull request:
EDIT: This pull request is getting pretty big, it contains the following fixes because you can't really choose what goes into a pull request, and multiple edits all edit minecraft.yaml. It's all been tested, dev build available at https://github.com/Khroki/mcedit/releases/tag/v1.0.2
-Quartz and sandstone stairs texture fix
-1.8 Prerelease block updates
-Sapling coord texture fix
-OniOni/hetmankp bugfixes
-Slab rename
-Renderer.py adjustments
-Enaut's python 2.6 compatibility patch
-Aiybe's redstone light fix
-Fixes opaque stained glass

Khroki committed:
Missing tex fix.

Forgot the quartz stairs texture as it was showing correctly in my test build and 2 sapling coords were flipped, something about the sapling type coding is making them display incorrectly however, not really worried about it though. Sorry for all the minor fixes.

Lechros commented:
lol I made a terrain file with number on each blocks so that i dont have to count blocks everytime i write tex location

Khroki commented:
I was tired ok. :P

Khroki committed:
Minecraft.yaml 1.8 fix compliation

Credits to OniOniOn and hetmankp for finding these.
Added upper wooden slabs, fixed leaf and iron trapdoor naming discrepancies.

Khroki commented:
Committed some of the other minecraft.yaml fixes so I could use them in my build, if it get pulled thanks to onionion and hetmankp for noticing them. Also contains a few other minor things I noticed, there's always something.

Khroki committed:
Slab tweak

Old wood slab change to match the wiki.

Lechros commented:
Just to let you know...
Nobody changed Nether Brick Fence Render Shape for more than a year

Khroki commented:
Yeah, I noticed that, along with a few other blocks. I suspect they need to be added elsewhere in the code as both types of fences are set to FENCE but only one renders correctly. If I get the time I'll try and find it.

Lechros commented:
1 hour left until 0.9.0 blocks

Khroki committed:
Partial static block update

Added a few missing static block mappings for testing purposes

Khroki committed:
Enaut's python 2.6 compatibility patch

Khroki committed:
Redstone block fix

Huh, didn't even notice.

Khroki commented:
Kinda annoying every change I commit for personal use ends up on this pull request, makes it overlap with other requests.

dratini0 commented:
You can make a personal-use branch, if you wish.

Kovács Bálint

On Sun, Jun 15, 2014 at 10:39 PM, Khroki notifications@github.com wrote:

> Kinda annoying every change I commit for personal use ends up on this pull
> request, makes it overlap with other requests.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mcedit/pymclevel/pull/161#issuecomment-46127173.

Khroki commented:
That's probably a good idea.
Edit: done. I'll leave these edits on this request for now, they're pretty harmless.

Khroki committed:
14w25a Block update

Added new blocks

Khroki commented:
Now also includes 14w25a's new blocks, some random fixes, and support for my renderer.py update.

Khroki committed:
Minecraft.yaml+Materials.py update

various fixes and updates plus tweaks for updated renderer.py.

Khroki committed:
1.8 Prerelease Update

Added all the new blocks and fixed a few incorrect textures.

Khroki committed:
Red Sandstone Slab Fix

Forgot to add the upper version

Khroki committed:
1.8 release definitions update

Items.py needed an overhaul for 1.7+ compatibility.

kdart commented:
Will this be pulled?

Lechros commented:
Probably not.
